packages/react-router-dom/index.tsx

```diff
@@ -322,7 +323,7 @@ export const NavLink = React.forwardRef<HTMLAnchorElement, NavLinkProps>(
     }
 
     let isActive =
-      locationPathname === toPathname ||
+      !!usePath(toPathname) ||
```

Comment:
Do you mean `useResolvedPath` or `matchPath`? `usePath` isn't an API on react-router.

Reply:
Yeah I meant `useMatch`!